apis/datadoghq/common/const.go

```diff
@@ -161,6 +161,10 @@ const (
 	AgentCustomConfigVolumePath = "/etc/datadog-agent/datadog.yaml"
 	SystemProbeConfigVolumePath = "/etc/datadog-agent/system-probe.yaml"
 
+	// Service Internal Traffic Policy exists in Kube 1.21 but it is enabled by default since 1.22
+	LocalServiceMinimumVersion        = "1.21-0"
+	LocalServiceDefaultMinimumVersion = "1.22-0"
```

Comment:
do you anticipate these will be referenced elsewhere besides in `ShouldCreateAgentLocalService` ?

Reply:
I don't think so for now. I'll move them.